qucs/qucs/qucs.cpp

```diff
@@ -1226,7 +1226,8 @@ void QucsApp::slotListProjOpen(const QModelIndex &idx)
     // change projects directory to the selected one
     QucsSettings.projsDir = QucsSettings.projsDir.filePath(dName);
     readProjects();
-    slotUpdateTreeview();
+    // BUG?: it's no longer there.
```

Comment:
could as well have cherry-pick #871 here: `slotUpdateTreeview()` is no longer needed since the Project View now updates by itself whenever there's a change in the directory being watched. That function was removed in #770 but later #767 was merged and brought this call to `slotUpdateTreeview()` back; this because #767 should have been rebased before merging or merged before #770.
Note that I also commented out the call to `repaint()` since I did not see why it's needed and everything appears to work fine without.

Reply:
You are right.
I didn't do that because I thought we will merge 9afdd76 and 3b56529  (even if the build failed) and then merge #871